purchase_request/security/ir.model.access.csv

```diff
@@ -9,6 +9,6 @@ access_purchase_order_line_user,access.purchase.order.line.user,model_purchase_o
 access_purchase_order_manager,access.purchase.order.manager,model_purchase_order,purchase_request.group_purchase_request_manager,1,0,0,0
 access_purchase_order_line_manager,access.purchase.order.line.manager,model_purchase_order_line,purchase_request.group_purchase_request_manager,1,0,0,0
 access_purchase_request_allocation_manager,purchase.request.allocation,model_purchase_request_allocation,group_purchase_request_manager,1,1,1,1
-access_purchase_request_allocation_user,purchase.request.allocation,model_purchase_request_allocation,base.group_user,1,0,0,0
-access_purchase_request_line_stock_user,purchase.request.line.stock,model_purchase_request_line,stock.group_stock_user,1,0,0,0
+access_purchase_request_allocation_user,purchase.request.allocation,model_purchase_request_allocation,base.group_user,1,1,1,0
+access_purchase_request_line_stock_user,purchase.request.line.stock,model_purchase_request_line,stock.group_stock_user,1,1,1,0
```

Comment:
to revert again in v13

Reply:
For the bless of the new compute fields calculation :hugs: